request.js

```diff
@@ -116,15 +116,15 @@
         return callbacks;
     };
 
-    exports['get'] = function (url, query) {
+    exports.get = function (url, query) {
```

Comment:
If we use `exports.` we can export the delete method because delete is a reserved word in javascript, so I prefer to always (for this object) return the functions with `exports[]`